druid/src/widget/svg.rs

```diff
@@ -85,13 +88,29 @@ impl<T: Data> Widget<T> for Svg<T> {
     }
     fn paint(&mut self, paint_ctx: &mut PaintCtx, _data: &T, _env: &Env) {
         //TODO: options for aspect ratio or scaling based on height
-        let scale = paint_ctx.size().width / self.get_size().width;
+        let scalex = paint_ctx.size().width / self.get_size().width;
+        let scaley = paint_ctx.size().height / self.get_size().height;
+        let scale = scalex.min(scaley);
 
         let origin_x = (paint_ctx.size().width - (self.get_size().width * scale)) / 2.0;
         let origin_y = (paint_ctx.size().height - (self.get_size().height * scale)) / 2.0;
         let origin = Point::new(origin_x, origin_y);
 
-        self.svg_data.to_piet(scale, origin, paint_ctx);
+        let clip_rect = RoundedRect::from_origin_size(
```

Comment:
Is there a reason this is a `RoundedRect` instead of a plain `Rect`?

Reply:
Nope, I copy and pasted most of this from one of the other widgets, ill fix this up